.rebase/replace/code/src/vs/workbench/contrib/webview/browser/pre/index.html.json

```diff
@@ -0,0 +1,6 @@
+[
+	{
+		"from": "<meta http-equiv=\"Content-Security-Policy\"\\\n",
+		"by": "--->"
+	}
```

Comment:
the corresponding handler should be added to the `rebase.sh` script to get it working